### ISSUE TITLE
The Workspace Info Panel is empty

### ISSUE THREAD

**Issue by pcardona34** [trunkmaster/nextspace#449]
Hi Sergii,

This is a minor issue:
The Workspace Info Panel is empty.

I think it could be usefull to have these infos to get a full context when reporting an issue.

 - Context:
> 12.8
> Linux NEXTPI 6.6.51+rpt-rpi-v8 #1 SMP PREEMPT Debian 1:6.6.51-1+rpt3 (2024-10-08) aarch64 GNU/Linux

**Comment by trunkmaster** [comment]
Workspace Info panel mostly uses standard GNUstep routines. Should work without a problem.
Do you have any messages in console on first open of panel?

**Comment by trunkmaster** [comment]
@pcardona34 is it still actual?

**Comment by pcardona34** [comment]
Yes, it is actual: tested under Debian 12, aarch64 (last install today).
Nothing in the Console Log.

**Comment by pcardona34** [comment]
The same under Ultramarine (Fedora like) so it seems a specific issues of the Raspberries (maybe a device-tree trap) or of the arm kernel. I will investigate.

**Comment by trunkmaster** [comment]
I've mananged to reproduce this on VM on MacBook Air M2. It seems that it's a GNUstep bug.

**Comment by pcardona34** [comment]
In the case of my pi400, `lscpu` is verbose...

[cpuinfos_pi400.txt](https://github.com/user-attachments/files/19276255/cpuinfos_pi400.txt)

**Comment by pcardona34** [comment]
Well done. Maybe later we could get the CPU info on arm machines.

![Image](https://github.com/user-attachments/assets/9e2d0f96-21ba-4e79-b1a1-93c138cc9264)
